README.md

```diff
@@ -115,12 +115,21 @@ These resources primarily work by writing the Sensu 2.x object definitions to a
 
 ## Resource Details
 
+### Common properties
+Sensu resources that support metadata attributes share these common properties:
+
+* `namespace` the Sensu RBAC namespace that this check belongs to, default: *default*
+* `labels` custom extended attributes to add to the check
```

Comment:
I think I should have a conversation with sensu about this from the description in the docs `annotations` seems like a neutered `labels` and I think this will lead to confusion.

Reply:
Definitely open to a discussion to understand design and intention better, but it's already been implemented upstream: https://github.com/sensu/sensu-go/pull/2283 so I figured we need to at least keep pace with features here so this cookbook doesn't sit in a broken state for long

Reply:
Let’s avoid using a word that makes it sound like things without testes are less than other things.

https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/

Reply:
@majormoses can you clarify what change you'd like me to make for this, happy adjust. Otherwise we can resolve?

Reply:
even after reading about kubernetes annotations and labels this is not quite clear on the differences in sensu and why it makes sense. I will accept this as its done but there is a documentation issue here. I don't think we should assume that everyone knows kubernetes and even so we still need to explain how ours differ from theirs as I am sure there are some differences. All I really grok'd from that is internal vs external metadata.